### ISSUE TITLE
AS3 Schema issue where it applies incorrect restriction if serverType is undefined

### ISSUE THREAD

**Issue by hshan278** [F5Networks/f5-appsvcs-extension#530]
Logging issue in Git per mail thread with Daniel. S

### Environment
 * Application Services Version: Unclear, some version prior to  3.31 but it likely happens there too
 * BIG-IP Version:  14.1.4.1 Build 0.0.4 Point Release 1

### Summary
When we use BIG-IQ to trigger AS3 to BIG-IP  to create 2 devices under same GSLB_Server we see an error '
    .com/devices: should NOT have more than 1 items'

Per debugging with BIG-IQ team and discussion with Daniel.S the root cause seems that there is some issue with AS3 schema
        --> "Looks like there is a bug in the schema that ends up applying the same restriction if serverType is undefined. Please file a bug on AS3's GitHub page: https://github.com/F5Networks/f5-appsvcs-extension/issues"

Below doc shows that 'serverType' property under GSLB_Server should default to 'bigip' 
    - https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/refguide/schemaref/GSLB_Server.schema.json.html?highlight=gslb_server
The doc also says that 'devices' can be an array, so multiple devices should work

NOTE:  A workaround for this issue is to explicitly  add ("serverType": "bigip") under ("class": "GSLB_Server") though documentation says that it is the default 

### Steps To Reproduce

1. SSH to BIG-IQ where a BIG-IP has been imported
2. Run below AS3 query on BIG-IQ which redirects it to BIG-IP
   **$ curl -s -u admin:f5-bigip01 http://localhost:8100/mgmt/shared/appsvcs/declare -X POST --upload-file /var/tmp/2-Server-Fails.json | jq .**

 NOTE: If you run the same query by removing one of two server devices under the GSLB_Server then also it works fine. There is some issue when there are multiple devices under GSLB_Object
 NOTE: If we run an equivalent query with 2 devices directly on BIG-IP even then it works fine.

3.  Failing JSON (2-Server-Fails.json)
```json
{
  "id": "urn:uuid:158c4f1f-5a00-46ad-9f56-3a30e5656c05",
  "class": "ADC",
  "target": {
    "address": "x.x.x.x"
  },
  "label": "Test DNS config",
  "Common": {
    "class": "Tenant",
    "Shared": {
      "LAB": {
        "class": "GSLB_Data_Center"
      },
      "class": "Application",
      "template": "shared",
      "labve01-02.test.com": {
        "class": "GSLB_Server",
        "devices": [
          {
            "address": "1.1.1.4"
          },
          {
            "address": "1.1.1.5"
          }
        ],
        "dataCenter": {
          "use": "/Common/Shared/LAB"
        },
        "exposeRouteDomainsEnabled": true
      }
    }
  },
  "remark": "testing DNS config",
  "controls": {
    "class": "Controls",
    "userAgent": "BIG-IQ/8.1 Configured by API"
  },
  "updateMode": "selective",
  "schemaVersion": "3.28.0"
}
```

2. Observe the following error response:
```json
{
  "code": 422,
  "message": "status:422, body:{\"code\":422,\"errors\":[\"/Common/Shared/labve01-02.test.com/devices: should NOT have more than 1 items\"],\"declarationFullId\":\"\",\"message\":\"declaration is invalid\"}",
  "originalRequestBody": "{\"code\":422,\"errors\":[\"/Common/Shared/labve01-02.test.com/devices: should NOT have more than 1 items\"],\"declarationFullId\":\"\",\"message\":\"declaration is invalid\"}",
  "referer": "Unknown",
  "restOperationId": 192699425,
  "errorStack": [],
  "kind": ":resterrorresponse"

}
```

### Expected Behavior
No error

### Actual Behavior
We see error code 422

**Comment by dstokesf5** [comment]
Thank you for your feedback. This issue has been added to our internal product backlog as AUTOTOOL-2878.

**Comment by dstokesf5** [comment]
This has been resolved in the 3.35.0 release.
